web-api/src/business/useCaseHelper/generateChangeOfAddressHelper.ts

```diff
@@ -107,10 +107,15 @@ export const generateChangeOfAddressHelper = async ({
     applicationContext.logger.error(error);
   }
 
+  const NOTIFICATION_ACTION:
+    | 'user_contact_update_progress'
+    | 'admin_contact_update_progress' =
+    `${websocketMessagePrefix}_contact_update_progress`;
```

Comment:
Nitpick: Is it worth typing this once in `generateChangeOfAddress` and then importing?